config/config.exs

```diff
@@ -4,3 +4,7 @@ use Mix.Config
 
 config :ex_gram,
   token: {:system, "BOT_TOKEN"}
+
+config :acm,
```

Comment:
Maybe change the `:acm` to `:acm_bot` as @rockneurotiko suggested

Reply:
The change from `:acm` to `:acm_bot` is mandatory if you want this to work. In elixir this configuration is for `application` configuration, and `:acm` is not an application, so you will see a warning when you run it (like `:acm application doesn't exists`) and won't be able to retrieve the value.

Reply:
So `ExGram` starts an application with ![this](https://github.com/acmfi/acm_bot/blob/develop/lib/acm_bot/bot.ex#L5) name?

Reply:
Nope! `:acm_bot` is the name of **this** application: https://github.com/acmfi/acm_bot/blob/develop/mix.exs#L6